### PR TITLE
Found that some things shifted and added in the writeScript import

### PR DESCRIPTION
**Pull request by pdramos1**
This worked locally previously but apparently the world shifted.. writeScript import is now added so sovrn amp ad will work.

### CODE REVIEW

File: 3p/integration.js [hunk 1]
```diff
@@ -69,6 +69,7 @@ import {colombia} from '../ads/colombia';
 
 
 
+
```
Comment (dvoytenko) [inline 3p/integration.js]: Could you please revert this file?

File: ads/sovrn.js [hunk 2]
```diff
@@ -16,19 +16,21 @@
  * Existing sovrn customers feel free to contact amp-implementations@sovrn.com for assistance with setting up your amp-ad tagid
  * New customers please see www.sovrn.com to sign up and get started!
  *********
```
Comment (dvoytenko) [inline ads/sovrn.js]: Could you please move this paragraph out of license and into a separate comment?

File: ads/sovrn.js [hunk 3]
```diff
@@ -16,19 +16,21 @@
...
+ **/
 /**
  * @param {!Window} global
  * @param {!Object} data
  */
-export function sovrn(global, data) {
+ import {writeScript} from '../src/3p';
```
Comment (dvoytenko) [inline ads/sovrn.js]: `import` should go above the comment with `@param` since that jsdoc documents the function itself.

File: ads/sovrn.js [hunk 4]
```diff
@@ -16,19 +16,21 @@
...
+ export function sovrn(global, data) {
```
Comment (dvoytenko) [inline ads/sovrn.js]: It looks like the code indented an extra space from the left edge. Could you please shift it one space back?

**Comment by dvoytenko** [comment]
@pdramos1 Thanks! Could you please address a couple of comments so that I can merge?

**Comment by pdramos1** [comment]
I'll address the issues. :)

**Comment by pdramos1** [comment]
@dvoytenko, made the changes you requested.

**Comment by dvoytenko** [comment]
LGTM. Thanks!

**Comment by pdramos1** [comment]
Thanks!

**Pull request merged**